service/history/api/recordactivitytaskstarted/api.go

```diff
@@ -48,6 +49,17 @@ import (
 	"go.temporal.io/server/service/history/workflow"
 )
 
+type RejectCode int32
+
+const (
+	reject_code_undefined       RejectCode = 0
+	activity_accepted           RejectCode = 1
+	rejected_stamp              RejectCode = 2
+	rejected_with_error         RejectCode = 3
```

Comment:
nit: probably not necessary and can just use default value for RejectCode when returned error is not nil. Caller won't/not supposed to inspect the code when error is returned.